README.md

```diff
@@ -31,6 +31,28 @@ steps:
 - run: java -cp java HelloWorldApp
 ```
 
+**Adopt OpenJDK Hotspot**
+```yaml
+steps:
+- uses: actions/checkout@v2
+- uses: actions/setup-java@v2
+  with:
+    distribution: 'adopt-hotspot' # See 'Supported distributions' for available options
+    java-version: '11'
+- run: java -cp java HelloWorldApp
+```
+
+**Adopt OpenJDK OpenJ9**
+```yaml
+steps:
+- uses: actions/checkout@v2
+- uses: actions/setup-java@v2
+  with:
+    distribution: 'adopt-openj9' # See 'Supported distributions' for available options
+    java-version: '11'
+- run: java -cp java HelloWorldApp
+```
+
```

Comment:
Considering the fact that the single difference between these examples and `adopt` is `distribution` field, I think that it should be enough to add new options only to the table Supported distributions below.

Reply:
The same for `advanced-usage.md` page.